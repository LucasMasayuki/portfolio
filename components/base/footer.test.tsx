import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import Footer from './footer';

describe.concurrent('Footer component', () => {
  describe('when render', () => {
    it('should fill the footer with correct components', () => {
      // When rendered
      const { container } = render(<Footer />);

      // Then it should show all content
      expect(container.innerHTML).toMatchSnapshot();
    });
  });
});
