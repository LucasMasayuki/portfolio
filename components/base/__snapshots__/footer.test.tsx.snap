// Vitest Snapshot v1

exports[`Footer component > when render > should fill the footer with correct components 1`] = `"<footer class=\\"_footer_e9bad9\\"><div class=\\"MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-3ze8r2-MuiPaper-root\\"><div class=\\"MuiToolbar-root MuiToolbar-regular css-r6ewbb-MuiToolbar-root\\"><div class=\\"MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root\\"><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1fpckdt-MuiGrid-root\\"><h6 class=\\"MuiTypography-root MuiTypography-subtitle2 css-zdy50j-MuiTypography-root\\">Version 0.1.0</h6></div></div></div></div></footer>"`;
